src/utils/time.ts

```diff
@@ -154,22 +163,31 @@ export function localizeTimePart(value: string, part: TimePart, locale: string):
   if (!date) {
     return;
   }
-  const formatter = createLocaleDateTimeFormatter(locale);
+  const formatter = createLocaleDateTimeFormatter(locale, numberingSystem);
   const parts = formatter.formatToParts(date);
   return getLocalizedTimePart(part, parts);
 }
 
-export function localizeTimeString(value: string, locale = "en", includeSeconds = true): string {
+export function localizeTimeString(
+  value: string,
+  locale = "en",
+  numberingSystem: NumberingSystem,
```

Comment:
☝️ should we initialize this to `defaultNumberingSystem` here as well?

Reply:
Nice catch, it should actually be using `getSupportedNumberingSystem()` instead using a default value. I changed the time util before writing that function for the Input PR. I'll fix that.

Reply:
Yup, totally.  Does that mean the other utils should use `getSupportedNumberingSystem` also?

Reply:
Yeah I'll go through all the time utils and make sure they are using a supported numbering system

Reply:
👍 on using params object. Once a method goes beyond 2 or 3 params, it's a good time to think about switching the signature.

Reply:
Cool I went ahead and changed the signatures and used `getSupportedNumberingSystem()` instead of a default value